### DISCUSSION TITLE
Calls to firestore from withAuthUserTokenSSR are unauthenticated.

### DISCUSSION THREAD

**Discussion by Roee-Tsur** [gladly-team/next-firebase-auth#627] [Q&A]
Hey all,
I'm testing my app using this package on the firebase emulator suite.
I discovered that my security rules deny my requests because the requests that are sent are unauthenticated.

In my function, I only make the firestore call after I validate that the user in authenticated.

Am I missing anything? what can I do to fix this?

**Comment by kmjennison** [comment]
Possibly related: #411

**Reply by Roee-Tsur** [reply]
I'm not sure this is it.
It may be because I'm using the javascript client firestore SDK and not the admin SDK.
do you think that should solve it?

**Reply by kmjennison** [reply]
I'm not sure—unfortunately, it's tough for me to debug specific implementations. Please be sure to look through existing issues related to Firestore in case somebody's faced a similar problem, and you might consider providing a minimal reproduction in case others can help.
